scrapyrt/core.py

```diff
@@ -164,7 +164,7 @@ def _get_log_file_path(self):
         log_dir = os.path.join(self.log_dir, self.spider_name)
         if not os.path.exists(log_dir):
             os.makedirs(log_dir)
-        filename = datetime.datetime.now().isoformat() + '.log'
+        filename = datetime.datetime.now().strftime('%Y-%m-%dT%H;%M;%S.%f') + '.log'
```

Comment:
The current pull request uses ; due to their simularity to : but I'm open to change it to either - or .
The benefit of using something other than - such as ; or . is that it makes easier to replace characters within the filename to return to iso 8601 programatically.